pydm/main_window.py

```diff
@@ -269,7 +272,12 @@ def toggle_status_bar(self, checked):
         self.ui.statusbar.setHidden(not checked)
 
     def get_files_in_display(self):
-        _, extension = path.splitext(self.current_file())
+        try:
+            curr_file = self.current_file()
+        except IndexError:
```

Comment:
Can you add a `logger` message here? Uncomfortable with exception endpoints that are hard to track.

Reply:
Addressed on the latest commit.